react/ProductContextProvider.tsx

```diff
@@ -15,7 +15,11 @@ function useProductInState(product: MaybeProduct, dispatch: Dispatch<Actions>) {
   }, [product, dispatch])
 }
 
-function useSelectedItemFromId(skuId: string | undefined, dispatch: Dispatch<Actions>, product: MaybeProduct) {
+function useSelectedItemFromId(
+  skuId: string | undefined,
```

Comment:
```suggestion
  skuId?: string,
```

Reply:
In fact this is an invalid syntax, as the question mark would mean an optional argument, and there can't be optional arguments before required ones